package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "justice-ui-library",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
I think if this is a major release, the version should become `1.0.0` instead

Reply:
alright